src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -52,34 +52,26 @@ private class Searcher
                 }
             }
 
-            internal void Search()
+            internal async void Search()
```

Comment:
This is an event handler.  It is called directly into by a void method we have to implement as part of a VS interface.